CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportController.java

```diff
@@ -170,7 +171,12 @@ public void setClearManuallyAdded(final boolean b) {
     }
 
     public void setDestination(final ImportDestination<?> destination) {
-        this.currentDestination = destination;
+        if (destination != null) {
+            this.currentDestination = destination;
+        }
+        if (this.currentDestination == null) {
+            return;
```

Comment:
sonar will probably flag this return as unnecessary

Reply:
Hi @CrucisGamma so @antares1470 has been smashing through SQ rules so any assistance to keep code smells down is greatly appreciated. What i'll do is merge it into a temporary branch  to see what SQ has to say. SQ can't run on your fork.